Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1375,11 +1403,32 @@ def run_polling_command(client, args: dict, cmd: str, upload_function: Callable,
             timeout_in_seconds=6000)
 
         command_result = CommandResults(scheduled_command=scheduled_command)
+    elif post_function:
+        # Validate the polling results
+        validate_submit_file_polling_results(command_result)
     return command_result
```

Comment:
```suggestion
    elif post_function:
        # Validate the polling results
    return  post_function(command_result) if post_function is not None else command_result
```

as discussed, we want to make sure we use the post function when needed and keep it generic as possible, without hurting other functionality.

Reply:
My bad, I meant to reference `post_function`.

Reply:
I made it as such:

`elif post_function is not None:`
`    # Validate the polling results`
`    post_function(command_result)`
`return command_result`

just because `post_function` does not return anything, it only validates the content of the polling results, and if there is an error in the report, then it raises an exception, otherwise, it does not return anything.